.github/workflows/ci.yml

```diff
@@ -256,5 +256,5 @@ jobs:
         if: failure() && inputs.failure-upload-path != ''
         with:
           name: failure-verify-${{ inputs.ff-os }}-${{ inputs.ff-jdk }}-${{ inputs.ff-jdk-distribution }}
-          path: ${{ inputs.failure-upload-path }}
+          path: 'target/*'
```

Comment:
I think we should change the default value instead, see above: https://github.com/killbill/gh-actions-shared/pull/36/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR110